Makefile: pin gomobile version to 5d9a3325

jakubgs opened the pull request:
Using latest version every time is just asking for weird random issues.

Also this commit has made Go `1.16` required: https://github.com/golang/mobile/commit/80cb72e7af8e71b65e7d9e59fb4ec5d620134608

Also refactored the `Makefile` a bit so install targets start with `install-*` for easier tab completion:
```
 > make help | grep --color=never -E '(install|setup)'
install:
  install-os-deps                 Operating System Dependencies
  install-gomobile                Go Mobile Build Tools
  install-lint                    Install Linting Tools
  install-mock                    Install Module Mocking Tools
  install-modvendor               Install Module Vendoring Tool
  install-protobuf                Install Protobuf Generation Tools
  install-release                 Install Github Release Tools
  install-xtools                  Install Miscellaneous Go Tools
setup:
  setup                           Install all tools
  setup-dev                       Install all necessary tools for development
  setup-build                     Install all necessary build tools
```

ghost commented:
<!--prchecklist--> 
### Pull Request Checklist
- [ ] Have you updated the documentation, if impacted (e.g. [docs.status.im](https://docs.status.im/docs/build_status_go.html))?
- [ ] Have you tested changes with mobile or desktop version?

status-im-auto commented:
### Jenkins Builds
<details>
<summary>Click to see older builds (142)</summary>

| :grey_question: | Commit | :hash: | Finished (UTC) | Duration | Platform | Result |
|-|-|-|-|-|-|-|
| :x: | 5fc3e4df | [#1](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-2352/1/) | 2021-09-07 12:46:46 | ~33 sec | `android` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-2352/1/consoleText) |
| :x: | 5fc3e4df | [#1](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-2352/1/) | 2021-09-07 12:46:59 | ~41 sec | `ios` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-2352/1/consoleText) |
| :heavy_check_mark: | 5fc3e4df | [#1](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-2352/1/) | 2021-09-07 12:48:30 | ~2 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-210907-124618-5fc3e4.zip) |
| | | | | | | |
| :x: | 93ac903a | [#2](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-2352/2/) | 2021-09-08 06:18:40 | ~49 sec | `ios` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-2352/2/consoleText) |
| :x: | 93ac903a | [#2](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-2352/2/) | 2021-09-08 06:18:48 | ~56 sec | `android` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-2352/2/consoleText) |
| :heavy_check_mark: | 93ac903a | [#2](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-2352/2/) | 2021-09-08 06:20:46 | ~2 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-210908-061751-93ac90.zip) |
| | | | | | | |
| :x: | dc83bece | [#3](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-2352/3/) | 2021-09-08 06:22:50 | ~14 sec | `android` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-2352/3/consoleText) |
| :x: | dc83bece | [#3](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-2352/3/) | 2021-09-08 06:22:51 | ~14 sec | `linux` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-2352/3/consoleText) |
| :x: | dc83bece | [#3](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-2352/3/) | 2021-09-08 06:22:52 | ~15 sec | `ios` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-2352/3/consoleText) |
| | | | | | | |
| :x: | e00713bc | [#4](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-2352/4/) | 2021-09-08 06:24:20 | ~13 sec | `ios` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-2352/4/consoleText) |
| :x: | e00713bc | [#4](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-2352/4/) | 2021-09-08 06:24:27 | ~20 sec | `android` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-2352/4/consoleText) |
| :heavy_check_mark: | e00713bc | [#4](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-2352/4/) | 2021-09-08 06:25:47 | ~1 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-210908-062406-e00713.zip) |
| | | | | | | |
| :x: | ae8a2754 | [#5](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-2352/5/) | 2021-09-08 06:26:45 | ~23 sec | `ios` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-2352/5/consoleText) |
| :x: | ae8a2754 | [#5](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-2352/5/) | 2021-09-08 06:26:48 | ~27 sec | `android` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-2352/5/consoleText) |
| :heavy_check_mark: | ae8a2754 | [#5](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-2352/5/) | 2021-09-08 06:28:45 | ~2 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-210908-062621-ae8a27.zip) |
| | | | | | | |
| :x: | ae9060fa | [#6](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-2352/6/) | 2021-09-08 06:30:45 | ~18 sec | `ios` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-2352/6/consoleText) |
| :x: | ae9060fa | [#6](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-2352/6/) | 2021-09-08 06:30:49 | ~22 sec | `android` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-2352/6/consoleText) |
| :heavy_check_mark: | ae9060fa | [#6](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-2352/6/) | 2021-09-08 06:32:16 | ~1 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-210908-063026-ae9060.zip) |
| | | | | | | |
| :x: | b8114ad6 | [#7](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-2352/7/) | 2021-09-08 06:40:09 | ~22 sec | `ios` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-2352/7/consoleText) |
| :x: | b8114ad6 | [#7](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-2352/7/) | 2021-09-08 06:40:11 | ~24 sec | `android` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-2352/7/consoleText) |
| :heavy_check_mark: | b8114ad6 | [#7](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-2352/7/) | 2021-09-08 06:42:58 | ~3 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-210908-063946-b8114a.zip) |
| | | | | | | |
| :x: | 6fa4d42e | [#8](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-2352/8/) | 2021-09-08 06:52:59 | ~21 sec | `ios` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-2352/8/consoleText) |
| :x: | 6fa4d42e | [#8](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-2352/8/) | 2021-09-08 06:53:24 | ~47 sec | `android` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-2352/8/consoleText) |
| :heavy_check_mark: | 6fa4d42e | [#8](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-2352/8/) | 2021-09-08 06:54:23 | ~1 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-210908-065236-6fa4d4.zip) |
| | | | | | | |
| :x: | 4b6c9f70 | [#9](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-2352/9/) | 2021-09-08 07:08:28 | ~21 sec | `ios` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-2352/9/consoleText) |
| :x: | 4b6c9f70 | [#9](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-2352/9/) | 2021-09-08 07:08:29 | ~22 sec | `android` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-2352/9/consoleText) |
| :heavy_check_mark: | 4b6c9f70 | [#9](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-2352/9/) | 2021-09-08 07:09:44 | ~1 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-210908-070806-4b6c9f.zip) |
| | | | | | | |
| :x: | 367339d2 | [#10](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-2352/10/) | 2021-09-08 07:12:21 | ~19 sec | `ios` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-2352/10/consoleText) |
| :x: | 367339d2 | [#10](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-2352/10/) | 2021-09-08 07:12:24 | ~20 sec | `android` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-2352/10/consoleText) |
| :heavy_check_mark: | 367339d2 | [#10](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-2352/10/) | 2021-09-08 07:13:36 | ~1 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-210908-071202-367339.zip) |
| | | | | | | |
| :x: | 3021394c | [#11](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-2352/11/) | 2021-09-08 07:23:03 | ~25 sec | `ios` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-2352/11/consoleText) |
| :x: | 3021394c | [#11](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-2352/11/) | 2021-09-08 07:23:28 | ~51 sec | `android` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-2352/11/consoleText) |
| :heavy_check_mark: | 3021394c | [#11](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-2352/11/) | 2021-09-08 07:25:10 | ~2 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-210908-072242-302139.zip) |
| :x: | 3021394c | [#13](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-2352/13/) | 2021-09-08 09:07:33 | ~19 sec | `ios` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-2352/13/consoleText) |
| :x: | 3021394c | [#13](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-2352/13/) | 2021-09-08 09:08:01 | ~48 sec | `android` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-2352/13/consoleText) |
| :heavy_check_mark: | 3021394c | [#13](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-2352/13/) | 2021-09-08 09:09:12 | ~1 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-210908-090712-302139.zip) |
| | | | | | | |
| :x: | 72407626 | [#12](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-2352/12/) | 2021-09-08 07:26:21 | ~23 sec | `android` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-2352/12/consoleText) |
| :x: | 72407626 | [#12](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-2352/12/) | 2021-09-08 07:26:25 | ~22 sec | `ios` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-2352/12/consoleText) |
| :heavy_check_mark: | 72407626 | [#12](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-2352/12/) | 2021-09-08 07:28:27 | ~2 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-210908-072602-724076.zip) |
| | | | | | | |
| :x: | 3afb27ca | [#14](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-2352/14/) | 2021-09-08 09:12:52 | ~52 sec | `android` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-2352/14/consoleText) |
| :x: | 3afb27ca | [#14](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-2352/14/) | 2021-09-08 09:13:07 | ~1 min | `ios` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-2352/14/consoleText) |
| :heavy_check_mark: | 3afb27ca | [#14](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-2352/14/) | 2021-09-08 09:14:25 | ~2 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-210908-091202-3afb27.zip) |
| :x: | 3afb27ca | [#15](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-2352/15/) | 2021-09-08 09:47:33 | ~1 min | `android` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-2352/15/consoleText) |
| | | | | | | |
| :x: | bbc29e61 | [#16](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-2352/16/) | 2021-09-08 10:07:20 | ~21 sec | `android` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-2352/16/consoleText) |
| :x: | bbc29e61 | [#15](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-2352/15/) | 2021-09-08 10:07:23 | ~20 sec | `ios` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-2352/15/consoleText) |
| :heavy_check_mark: | bbc29e61 | [#15](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-2352/15/) | 2021-09-08 10:08:47 | ~1 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-210908-100702-bbc29e.zip) |
| | | | | | | |
| :x: | 464f2833 | [#16](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-2352/16/) | 2021-09-08 10:11:47 | ~18 sec | `ios` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-2352/16/consoleText) |
| :x: | 464f2833 | [#17](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-2352/17/) | 2021-09-08 10:12:14 | ~46 sec | `android` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-2352/17/consoleText) |
| :heavy_check_mark: | 464f2833 | [#16](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-2352/16/) | 2021-09-08 10:13:06 | ~1 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-210908-101128-464f28.zip) |
| | | | | | | |
| :x: | 29a406f0 | [#18](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-2352/18/) | 2021-09-08 10:16:49 | ~21 sec | `android` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-2352/18/consoleText) |
| :x: | 29a406f0 | [#17](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-2352/17/) | 2021-09-08 10:16:52 | ~23 sec | `ios` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-2352/17/consoleText) |
| :heavy_check_mark: | 29a406f0 | [#17](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-2352/17/) | 2021-09-08 10:18:48 | ~2 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-210908-101633-29a406.zip) |
| | | | | | | |
| :x: | 0a170a70 | [#19](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-2352/19/) | 2021-09-08 10:28:21 | ~30 sec | `android` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-2352/19/consoleText) |
| :heavy_check_mark: | 0a170a70 | [#18](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-2352/18/) | 2021-09-08 10:29:52 | ~1 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-210908-102753-0a170a.zip) |
| :heavy_check_mark: | 0a170a70 | [#18](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-2352/18/) | 2021-09-08 10:33:07 | ~5 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-210908-102750-0a170a.zip) |
| | | | | | | |
| :x: | 6cf8aef9 | [#20](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-2352/20/) | 2021-09-08 10:30:48 | ~36 sec | `android` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-2352/20/consoleText) |
| :heavy_check_mark: | 6cf8aef9 | [#19](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-2352/19/) | 2021-09-08 10:31:51 | ~1 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-210908-103013-6cf8ae.zip) |
| :heavy_check_mark: | 6cf8aef9 | [#19](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-2352/19/) | 2021-09-08 10:36:33 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-210908-103309-6cf8ae.zip) |
| | | | | | | |
| :x: | ba3ed5db | [#21](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-2352/21/) | 2021-09-08 10:33:53 | ~20 sec | `android` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-2352/21/consoleText) |
| :heavy_check_mark: | ba3ed5db | [#20](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-2352/20/) | 2021-09-08 10:35:26 | ~1 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-210908-103334-ba3ed5.zip) |
| :heavy_check_mark: | ba3ed5db | [#20](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-2352/20/) | 2021-09-08 10:40:31 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-210908-103635-ba3ed5.zip) |
| | | | | | | |
| :x: | a4b04ed4 | [#22](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-2352/22/) | 2021-09-08 10:42:33 | ~17 sec | `android` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-2352/22/consoleText) |
| :x: | a4b04ed4 | [#21](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-2352/21/) | 2021-09-08 10:42:41 | ~21 sec | `ios` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-2352/21/consoleText) |
| :heavy_check_mark: | a4b04ed4 | [#21](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-2352/21/) | 2021-09-08 10:44:25 | ~2 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-210908-104218-a4b04e.zip) |
| | | | | | | |
| :x: | 7c0481f9 | [#22](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-2352/22/) | 2021-09-08 10:44:13 | ~29 sec | `ios` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-2352/22/consoleText) |
| :x: | 7c0481f9 | [#23](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-2352/23/) | 2021-09-08 10:44:19 | ~37 sec | `android` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-2352/23/consoleText) |
| :heavy_check_mark: | 7c0481f9 | [#22](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-2352/22/) | 2021-09-08 10:46:26 | ~1 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-210908-104428-7c0481.zip) |
| | | | | | | |
| :x: | 38e091de | [#23](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-2352/23/) | 2021-09-09 08:51:50 | ~21 sec | `ios` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-2352/23/consoleText) |
| :x: | 38e091de | [#24](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-2352/24/) | 2021-09-09 08:51:55 | ~26 sec | `android` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-2352/24/consoleText) |
| :heavy_check_mark: | 38e091de | [#23](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-2352/23/) | 2021-09-09 08:53:50 | ~2 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-210909-085128-38e091.zip) |
| | | | | | | |
| :x: | 49fd0eed | [#25](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-2352/25/) | 2021-09-09 09:13:06 | ~25 sec | `android` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-2352/25/consoleText) |
| :x: | 49fd0eed | [#24](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-2352/24/) | 2021-09-09 09:13:06 | ~25 sec | `ios` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-2352/24/consoleText) |
| :heavy_check_mark: | 49fd0eed | [#24](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-2352/24/) | 2021-09-09 09:14:41 | ~1 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-210909-091244-49fd0e.zip) |
| | | | | | | |
| :x: | 87f5ae94 | [#25](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-2352/25/) | 2021-09-09 09:31:10 | ~52 sec | `ios` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-2352/25/consoleText) |
| :x: | 87f5ae94 | [#26](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-2352/26/) | 2021-09-09 09:31:36 | ~1 min | `android` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-2352/26/consoleText) |
| :heavy_check_mark: | 87f5ae94 | [#25](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-2352/25/) | 2021-09-09 09:33:43 | ~3 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-210909-093016-87f5ae.zip) |
| | | | | | | |
| :x: | 457331b4 | [#27](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-2352/27/) | 2021-09-09 09:49:17 | ~3 min | `android` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-2352/27/consoleText) |
| :heavy_check_mark: | 457331b4 | [#26](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-2352/26/) | 2021-09-09 09:50:51 | ~4 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-210909-094557-457331.zip) |
| :x: | 457331b4 | [#26](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-2352/26/) | 2021-09-09 09:51:13 | ~5 min | `ios` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-2352/26/consoleText) |
| | | | | | | |
| :x: | b8844404 | [#27](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-2352/27/) | 2021-09-09 10:00:14 | ~22 sec | `ios` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-2352/27/consoleText) |
| :x: | b8844404 | [#28](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-2352/28/) | 2021-09-09 10:00:23 | ~31 sec | `android` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-2352/28/consoleText) |
| :heavy_check_mark: | b8844404 | [#27](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-2352/27/) | 2021-09-09 10:01:58 | ~2 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-210909-095954-b88444.zip) |
| | | | | | | |
| :x: | 7e4b2942 | [#28](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-2352/28/) | 2021-09-09 10:17:31 | ~35 sec | `ios` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-2352/28/consoleText) |
| :x: | 7e4b2942 | [#29](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-2352/29/) | 2021-09-09 10:17:34 | ~38 sec | `android` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-2352/29/consoleText) |
| :heavy_check_mark: | 7e4b2942 | [#28](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-2352/28/) | 2021-09-09 10:19:16 | ~2 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-210909-101655-7e4b29.zip) |
| | | | | | | |
| :x: | 132efdb8 | [#29](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-2352/29/) | 2021-09-09 10:20:47 | ~16 sec | `ios` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-2352/29/consoleText) |
| :x: | 132efdb8 | [#30](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-2352/30/) | 2021-09-09 10:20:48 | ~18 sec | `android` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-2352/30/consoleText) |
| :heavy_check_mark: | 132efdb8 | [#29](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-2352/29/) | 2021-09-09 10:22:50 | ~2 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-210909-102030-132efd.zip) |
| | | | | | | |
| :x: | 99c0aba1 | [#31](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-2352/31/) | 2021-09-09 10:37:18 | ~32 sec | `android` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-2352/31/consoleText) |
| :heavy_check_mark: | 99c0aba1 | [#30](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-2352/30/) | 2021-09-09 10:38:28 | ~1 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-210909-103645-99c0ab.zip) |
| :heavy_check_mark: | 99c0aba1 | [#30](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-2352/30/) | 2021-09-09 10:39:48 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-210909-103645-99c0ab.zip) |
| | | | | | | |
| :heavy_check_mark: | 39798307 | [#31](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-2352/31/) | 2021-09-09 10:53:54 | ~2 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-210909-105139-397983.zip) |
| :heavy_check_mark: | 39798307 | [#31](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-2352/31/) | 2021-09-09 10:54:33 | ~2 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-210909-105139-397983.zip) |
| :heavy_check_mark: | 39798307 | [#32](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-2352/32/) | 2021-09-09 10:58:08 | ~6 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-210909-105136-397983.aar) |
| | | | | | | |
| :x: | c2376f71 | [#33](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-2352/33/) | 2021-09-09 19:27:22 | ~27 sec | `android` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-2352/33/consoleText) |
| :x: | c2376f71 | [#32](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-2352/32/) | 2021-09-09 19:27:22 | ~27 sec | `ios` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-2352/32/consoleText) |
| :heavy_check_mark: | c2376f71 | [#32](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-2352/32/) | 2021-09-09 19:29:39 | ~2 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-210909-192654-c2376f.zip) |
| | | | | | | |
| :heavy_check_mark: | db8a4ff7 | [#33](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-2352/33/) | 2021-09-09 19:48:30 | ~2 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-210909-194625-db8a4f.zip) |
| :heavy_check_mark: | db8a4ff7 | [#33](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-2352/33/) | 2021-09-09 19:49:19 | ~2 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-210909-194625-db8a4f.zip) |
| :heavy_check_mark: | db8a4ff7 | [#34](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-2352/34/) | 2021-09-09 19:53:04 | ~6 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-210909-194620-db8a4f.aar) |
| | | | | | | |
| :x: | 60b35e91 | [#35](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-2352/35/) | 2021-09-10 07:10:33 | ~25 sec | `android` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-2352/35/consoleText) |
| :heavy_check_mark: | 60b35e91 | [#34](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-2352/34/) | 2021-09-10 07:12:46 | ~2 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-210910-071012-60b35e.zip) |
| :heavy_check_mark: | 60b35e91 | [#34](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-2352/34/) | 2021-09-10 07:19:56 | ~9 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-210910-071012-60b35e.zip) |
| | | | | | | |
| :x: | 90c562f0 | [#36](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-2352/36/) | 2021-09-10 07:12:23 | ~25 sec | `android` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-2352/36/consoleText) |
| :heavy_check_mark: | 90c562f0 | [#35](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-2352/35/) | 2021-09-10 07:14:30 | ~1 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-210910-071248-90c562.zip) |
| :heavy_check_mark: | 90c562f0 | [#35](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-2352/35/) | 2021-09-10 07:29:37 | ~9 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-210910-071958-90c562.zip) |
| | | | | | | |
| :x: | 5e1d2e96 | [#37](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-2352/37/) | 2021-09-10 07:21:24 | ~26 sec | `android` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-2352/37/consoleText) |
| :heavy_check_mark: | 5e1d2e96 | [#36](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-2352/36/) | 2021-09-10 07:22:57 | ~1 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-210910-072057-5e1d2e.zip) |
| :heavy_check_mark: | 5e1d2e96 | [#36](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-2352/36/) | 2021-09-10 07:42:42 | ~13 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-210910-072939-5e1d2e.zip) |
| | | | | | | |
| :x: | f84d701f | [#38](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-2352/38/) | 2021-09-10 09:09:09 | ~30 sec | `android` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-2352/38/consoleText) |
| :heavy_check_mark: | f84d701f | [#37](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-2352/37/) | 2021-09-10 09:10:51 | ~2 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-210910-090842-f84d70.zip) |
| :heavy_check_mark: | f84d701f | [#37](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-2352/37/) | 2021-09-10 09:19:44 | ~11 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-210910-090839-f84d70.zip) |
| | | | | | | |
| :x: | bb3b665d | [#39](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-2352/39/) | 2021-09-10 09:12:15 | ~25 sec | `android` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-2352/39/consoleText) |
| :heavy_check_mark: | bb3b665d | [#38](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-2352/38/) | 2021-09-10 09:14:16 | ~2 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-210910-091149-bb3b66.zip) |
| :x: | bb3b665d | [#38](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-2352/38/) | 2021-09-10 09:20:07 | ~19 sec | `ios` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-2352/38/consoleText) |
| | | | | | | |
| :x: | 82bf3cce | [#39](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-2352/39/) | 2021-09-10 09:23:35 | ~19 sec | `ios` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-2352/39/consoleText) |
| :x: | 82bf3cce | [#40](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-2352/40/) | 2021-09-10 09:23:36 | ~20 sec | `android` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-2352/40/consoleText) |
| :heavy_check_mark: | 82bf3cce | [#39](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-2352/39/) | 2021-09-10 09:25:04 | ~1 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-210910-092315-82bf3c.zip) |
| | | | | | | |
| :x: | 710ba4bd | [#40](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-2352/40/) | 2021-09-13 09:07:50 | ~39 sec | `ios` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-2352/40/consoleText) |
| :x: | 710ba4bd | [#40](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-2352/40/) | 2021-09-13 09:07:57 | ~45 sec | `linux` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-2352/40/consoleText) |
| :x: | 710ba4bd | [#41](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-2352/41/) | 2021-09-13 09:08:30 | ~1 min | `android` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-2352/41/consoleText) |
| | | | | | | |
| :x: | 313cca20 | [#41](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-2352/41/) | 2021-09-13 09:10:18 | ~35 sec | `ios` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-2352/41/consoleText) |
| :x: | 313cca20 | [#42](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-2352/42/) | 2021-09-13 09:10:30 | ~48 sec | `android` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-2352/42/consoleText) |
| :heavy_check_mark: | 313cca20 | [#41](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-2352/41/) | 2021-09-13 09:12:21 | ~2 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-210913-090946-313cca.zip) |
| | | | | | | |
| :x: | 40324b50 | [#42](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-2352/42/) | 2021-09-13 09:14:49 | ~26 sec | `ios` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-2352/42/consoleText) |
| :x: | 40324b50 | [#43](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-2352/43/) | 2021-09-13 09:15:04 | ~41 sec | `android` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-2352/43/consoleText) |
| :heavy_check_mark: | 40324b50 | [#42](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-2352/42/) | 2021-09-13 09:17:03 | ~2 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-210913-091427-40324b.zip) |
| | | | | | | |
| :x: | 92debd97 | [#43](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-2352/43/) | 2021-09-13 09:16:45 | ~23 sec | `ios` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-2352/43/consoleText) |
| :x: | 92debd97 | [#44](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-2352/44/) | 2021-09-13 09:17:04 | ~41 sec | `android` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-2352/44/consoleText) |
| :heavy_check_mark: | 92debd97 | [#43](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-2352/43/) | 2021-09-13 09:20:20 | ~3 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-210913-091706-92debd.zip) |
| | | | | | | |
| :x: | f0459642 | [#45](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-2352/45/) | 2021-09-13 09:18:51 | ~28 sec | `android` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-2352/45/consoleText) |
| :x: | f0459642 | [#44](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-2352/44/) | 2021-09-13 09:19:37 | ~1 min | `ios` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-2352/44/consoleText) |
| :heavy_check_mark: | f0459642 | [#44](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-2352/44/) | 2021-09-13 09:22:39 | ~2 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-210913-092024-f04596.zip) |
| | | | | | | |
| :x: | dc1bb9fa | [#46](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-2352/46/) | 2021-09-14 10:33:58 | ~22 sec | `android` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-2352/46/consoleText) |
| :x: | dc1bb9fa | [#45](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-2352/45/) | 2021-09-14 10:33:59 | ~22 sec | `linux` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-2352/45/consoleText) |
| :x: | dc1bb9fa | [#45](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-2352/45/) | 2021-09-14 10:34:00 | ~23 sec | `ios` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-2352/45/consoleText) |
| | | | | | | |
| :heavy_check_mark: | 70384cbb | [#46](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-2352/46/) | 2021-09-14 10:38:07 | ~2 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-210914-103526-70384c.zip) |
| :heavy_check_mark: | 70384cbb | [#46](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-2352/46/) | 2021-09-14 10:38:08 | ~2 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-210914-103526-70384c.zip) |
| :heavy_check_mark: | 70384cbb | [#47](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-2352/47/) | 2021-09-14 10:42:03 | ~6 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-210914-103521-70384c.aar) |
| | | | | | | |
| :heavy_check_mark: | 3ea641c5 | [#47](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-2352/47/) | 2021-09-14 10:45:10 | ~1 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-210914-104321-3ea641.zip) |
| :heavy_check_mark: | 3ea641c5 | [#47](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-2352/47/) | 2021-09-14 10:45:31 | ~2 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-210914-104321-3ea641.zip) |
| :heavy_check_mark: | 3ea641c5 | [#48](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-2352/48/) | 2021-09-14 10:50:43 | ~7 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-210914-104321-3ea641.aar) |
</details>

| :grey_question: | Commit | :hash: | Finished (UTC) | Duration | Platform | Result |
|-|-|-|-|-|-|-|
| :heavy_check_mark: | 9e06d50c | [#48](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-2352/48/) | 2021-09-14 11:44:02 | ~2 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-210914-114123-9e06d5.zip) |
| :heavy_check_mark: | 9e06d50c | [#48](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-2352/48/) | 2021-09-14 11:44:03 | ~2 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-210914-114123-9e06d5.zip) |
| :heavy_check_mark: | 9e06d50c | [#49](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-2352/49/) | 2021-09-14 11:47:27 | ~6 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-210914-114119-9e06d5.aar) |
| | | | | | | |
| :heavy_check_mark: | d0a9e113 | [#49](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-2352/49/) | 2021-09-14 12:10:39 | ~2 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-210914-120755-d0a9e1.zip) |
| :heavy_check_mark: | d0a9e113 | [#49](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-2352/49/) | 2021-09-14 12:10:45 | ~2 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-210914-120755-d0a9e1.zip) |
| :heavy_check_mark: | d0a9e113 | [#50](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-2352/50/) | 2021-09-14 12:13:33 | ~5 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-210914-120755-d0a9e1.aar) |

jakubgs commented:
What's weird is that when I run `make vendor` I get changes, so someone must have forgotten to run it:
```
 go.mod | 1 +
 go.sum | 7 ++++++-
 2 files changed, 7 insertions(+), 1 deletion(-
```
But when I run the build I also get failures because there are even more changes that show up in CI:

![image](https://user-images.githubusercontent.com/2212681/132480988-3c7ed661-1f19-43c6-be97-278e318d98fe.png)

jakubgs commented:
If I don't use `GO111MODULE=off` I get this:
```
go get -u github.com/c4milo/github-release
# special fix for gomobile issues
go get golang.org/x/tools/go/packages
go get golang.org/x/mobile/cmd/gomobile@5d9a3325
go: can only use path@version syntax with 'go get' and 'go install' in module-aware mode
make: *** [Makefile:243: gomobile-install] Error 1
```
But I see that `develop` builds fine: https://ci.status.im/job/status-go/job/manual/833/

jakubgs commented:
If I disable `GO111MODULE=off` in `Jenkinsfiles` I get this:
```
+ make statusgo-android
Building status-go for Android...
gomobile init
gomobile: go install golang.org/x/mobile/cmd/gobind failed: exit status 1
missing go.sum entry for module providing package golang.org/x/mobile/cmd/gobind; to add:
	go mod download golang.org/x/mobile

make: *** [Makefile:110: statusgo-android] Error 1
```

jakubgs commented:
We actually had some similar issues with `gomobile` before: https://github.com/status-im/status-go/issues/1573

jakubgs commented:
And here's yet another type of error with `GO111MODULE=on` for the `gomobile` comand...
```
+ make statusgo-android
Building status-go for Android...
gomobile init
gomobile: go install golang.org/x/mobile/cmd/gobind failed: exit status 1
cannot find package "golang.org/x/mobile/cmd/gobind" in any of:
	/usr/local/go/src/golang.org/x/mobile/cmd/gobind (from $GOROOT)
	/home/jenkins/workspace/status-go_prs_android_PR-2352/src/golang.org/x/mobile/cmd/gobind (from $GOPATH)

make: *** [Makefile:110: statusgo-android] Error 1
```
Quite a collection.

jakubgs commented:
This is interesting: https://golang.org/doc/go1.16#go-command

>Module-aware mode is enabled by default, regardless of whether a `go.mod` file is present in the current working directory or a parent directory. More precisely, the `GO111MODULE` environment variable now defaults to on. To switch to the previous behavior, set `GO111MODULE` to `auto`.
>
>Build commands like go build and go test no longer modify `go.mod` and `go.sum` by default. Instead, they report an error if a module requirement or checksum needs to be added or updated (as if the `-mod=readonly` flag were used). Module requirements and sums may be adjusted with go mod tidy or go get.
>
>`go install` now accepts arguments with version suffixes (for example, `go install example.com/cmd@v1.0.0`). This causes go install to build and install packages in module-aware mode, ignoring the go.mod file in the current directory or any parent directory, if there is one. This is useful for installing executables without affecting the dependencies of the main module.
>
>`go install`, with or without a version suffix (as described above), is now the recommended way to build and install packages in module mode. go get should be used with the `-d` flag to adjust the current module's dependencies without building packages, and use of go get to build and install packages is deprecated. In a future release, the `-d` flag will always be enabled.

Also this article is very helpful: https://maelvls.dev/go111module-everywhere/

jakubgs commented:
This is also a useful post: https://go.dev/blog/go116-module-changes

jakubgs commented:
I rewrote all tool installations to use `go install` and it works, but `modvendor` is giving me trouble:
```
 > go install github.com/adambabik/modvendor@5242804d
go install github.com/adambabik/modvendor@5242804d: github.com/adambabik/modvendor@none updating to
	github.com/adambabik/modvendor@v0.1.1-0.20200401195853-5242804d80d6: parsing go.mod:
	module declares its path as: github.com/goware/modvendor
	        but was required as: github.com/adambabik/modvendor
```
And I see why:
```go.mod
module github.com/goware/modvendor
```
https://github.com/goware/modvendor/blob/8f69eb91bc89d69d0d7381fb5b8ed25d21f0ff90/go.mod#L1

jakubgs commented:
I thought I could switch back to the origin of our `modvendor` fork, but they still haven't merged Adam's fix:
https://github.com/goware/modvendor/pull/13

So I've made a fork of Adams repo and fixed the module name: https://github.com/status-im/modvendor/commit/93d9b4a4a5ca2317f01bfee0d0649bff05823a45

jakubgs commented:
They install fine:
```
+ make setup-build modvendor-install
curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | BINARY=golangci-lint bash -s -- -d -b /home/jenkins/workspace/status-go_prs_android_PR-2352/bin v1.33.0
golangci/golangci-lint info checking GitHub for tag 'v1.33.0'
golangci/golangci-lint debug http_download https://github.com/golangci/golangci-lint/releases/v1.33.0
golangci/golangci-lint info found version: 1.33.0 for v1.33.0/linux/amd64
golangci/golangci-lint debug downloading files into /tmp/tmp.7iQDcWi5O3
golangci/golangci-lint debug http_download https://github.com/golangci/golangci-lint/releases/download/v1.33.0/golangci-lint-1.33.0-linux-amd64.tar.gz
golangci/golangci-lint debug http_download https://github.com/golangci/golangci-lint/releases/download/v1.33.0/golangci-lint-1.33.0-checksums.txt
golangci/golangci-lint info installed /home/jenkins/workspace/status-go_prs_android_PR-2352/bin/golangci-lint
GO111MODULE=on go install github.com/c4milo/github-release@v1.1.0
go: downloading github.com/c4milo/github-release v1.1.0
# special fix for gomobile issues
GO111MODULE=on go install golang.org/x/tools/go/packages/...@v0.1.5
go: downloading golang.org/x/tools v0.1.5
go: downloading golang.org/x/xerrors v0.0.0-20200804184101-5ec99f83aff1
go: downloading golang.org/x/sys v0.0.0-20210510120138-977fb7262007
go: downloading golang.org/x/mod v0.4.2
GO111MODULE=on go install golang.org/x/mobile/cmd/...@5d9a3325
go: downloading golang.org/x/mobile v0.0.0-20210902104108-5d9a33257ab5
go: downloading golang.org/x/tools v0.1.2
# a tool to vendor non-go files
# TODO: switch to original repo when https://github.com/goware/modvendor/pull/13 is merged
GO111MODULE=on go install github.com/status-im/modvendor@93d9b4a4
go: downloading github.com/status-im/modvendor v0.1.1-0.20210909091840-93d9b4a4a5ca
go: downloading github.com/mattn/go-zglob v0.0.0-20180803001819-2ea3427bfa53
```
But the build still fails:
```
+ make statusgo-android
Building status-go for Android...
gomobile init
gomobile: go install golang.org/x/mobile/cmd/gobind failed: exit status 1
cannot find package "golang.org/x/mobile/cmd/gobind" in any of:
	/usr/local/go/src/golang.org/x/mobile/cmd/gobind (from $GOROOT)
	/home/jenkins/workspace/status-go_prs_android_PR-2352/src/golang.org/x/mobile/cmd/gobind (from $GOPATH)

make: *** [Makefile:110: statusgo-android] Error 1
```
It seems like `gomobile` needs the `bind` command as a module and not as a tool.

jakubgs commented:
So it appears I get:
```
gomobile init
gomobile: go install golang.org/x/mobile/cmd/gobind failed: exit status 1
missing go.sum entry for module providing package golang.org/x/mobile/cmd/gobind; to add:
	go mod download golang.org/x/mobile
```
if I don't also do:
```
go get golang.org/x/mobile@5d9a3325
```
But if I do that and then run `make vendor` the `go.mod` and `go.sum` changes are reverted.

jakubgs commented:
How the HELL, is the error:
```
gomobile: go install golang.org/x/mobile/cmd/gobind failed: exit status 1
missing go.sum entry for module providing package golang.org/x/mobile/cmd/gobind; to add:
	go mod download golang.org/x/mobile
```
When it's clearly installed a moment earlier:
```
GO111MODULE=on go install golang.org/x/mobile/cmd/...@5d9a3325
GO111MODULE=on go get golang.org/x/mobile@5d9a3325
```
My best guess is that `make vendor` fucks things up.

jakubgs commented:
Yep, if I comment out the `make vendor` line then it works fine:

![image](https://user-images.githubusercontent.com/2212681/132673515-fa7459ee-158a-4d16-a3f3-40c71b0a676b.png)

I hate this vendor shit so much.

Samyoul commented on _assets/ci/Jenkinsfile.android:
```diff
@@ -59,8 +58,8 @@ pipeline {
       /* fail build if vendoring hasn't been done */
-      sh 'GO111MODULE=on make vendor'
-      sh 'git diff --exit-code --no-color --stat vendor/'
+      //sh 'GO111MODULE=on make vendor'
+      //sh 'git diff --exit-code --no-color --stat vendor/'
     } } }
```
It is good that this fixes the issue, I suppose this build would just crash if a future PR hasn't done the vendoring. ...

jakubgs replied:
Yeah. But that's not a proper fix. The `gomobile bind` command appears to depend on the `bind` module, not the `gobind` command, which is why it wants `go get golang.org/x/mobile`, but that changes `go.mod`, and running `make vendor` reverts that.

I have no idea what the proper fix for this is.

jakubgs replied:
I have an idea. I'll use `GO111MODULE=off` but only with the `go get golang.org/x/mobile` line.

jakubgs replied:
Ah, but that won't work...
```
GO111MODULE=off go get golang.org/x/mobile@5d9a3325
go: can only use path@version syntax with 'go get' and 'go install' in module-aware mode
make: *** [Makefile:244: gomobile-install] Error 1
```
For fuck's sake...

jakubgs replied:
These issues have been also reported here: https://github.com/golang/go/issues/46943

jakubgs replied:
Okay, I think I got it, and the secret sauce is this:
```Makefile
install-gomobile: ##@install Go Mobile Build Tools
	GO111MODULE=on  go install golang.org/x/mobile/cmd/...@5d9a3325
	GO111MODULE=on  go mod download golang.org/x/exp@ec7cb31e
	GO111MODULE=off go get -d golang.org/x/mobile/cmd/gobind
```

jakubgs commented:
I found this issue: https://github.com/golang/go/issues/44129

Apparently:

>Unfortunately I'm also hitting this error and neither `go get -t .` nor `go get ...` that `go1.16` suggests to do change anything and as the result: with `go1.16` build fails and cannot be fixed at all, while with `go1.15` build succeds ok. In other words it is `go1.16` regression that cannot be workerd around.

https://github.com/golang/go/issues/44129#issuecomment-788611690

And:

>This should be fixed in Go 1.17. __We don't plan to backport the fix to 1.16.x__; in order to avoid unintended regressions, we normally only backport fixes for security vulnerabilities and severe issues with no workaround.

https://github.com/golang/go/issues/44129#issuecomment-865249631

jakubgs committed:
Makefile: split commands into multiple lines

Signed-off-by: Jakub Sokołowski <jakub@status.im>

jakubgs committed:
Makefile: pin tool versions, use go install

Using latest version every time is just asking for weird random issues.

This commit in `gomobile` has made Go `1.16` required:
https://github.com/golang/mobile/commit/80cb72e7af8e71b65e7d9e59fb4ec5d620134608

Since we upgarded Go to 1.16 on CI hosts we can now use the recommended
method of installing tools, which is using `go install`.

In addition to that I've pinned other tools that we install to specific
versions, and dropped use of `modvendor` fork thanks to merging of:
https://github.com/goware/modvendor/pull/13

Signed-off-by: Jakub Sokołowski <jakub@status.im>

jakubgs committed:
install_deps.sh: fix format, add NixOS support

Signed-off-by: Jakub Sokołowski <jakub@status.im>
